cmd/virtlet_log/virtlet_log.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2017 Mirantis
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/Mirantis/virtlet/pkg/log"
+)
+
+/*
+* "virtlet_log" is an auxilary program that handles logs of VMs that are managed
+* by Virtlet. It is designed to continously run in its own Docker container on Kubernetes,
+* besides Virtlet container. It periodaically checks for new log folders and spawns
+* one worker for each log folder. Worker then "tail -F"s log file and converts each line
+* into JSON format that is expected by Kubernetes and lands output to the location where
+* Kubernetes expects it. This way "kubectl logs <VM>" suddenly starts working, as well as
+* the "Logs" tab on Kubernetes Dashboard.
+ */
+
+const (
+	DEFAULT_SLEEP_SECONDS = 10
+)
+
+func main() {
+	virtletFolder := os.Getenv("VIRTLET_VM_LOGS")
+	kubernetesFolder := os.Getenv("KUBERNETES_POD_LOGS")
+	if virtletFolder == "" || kubernetesFolder == "" {
+		fmt.Println("VIRTLET_VM_LOGS and KUBERNETES_POD_LOGS environment variables must be set")
+		os.Exit(-1)
+	}
+
+	sleepInterval := DEFAULT_SLEEP_SECONDS
+	if i, err := strconv.Atoi(os.Getenv("SLEEP_SECONDS")); err == nil {
+		sleepInterval = i
+	} else {
+		fmt.Println("failed to parse SLEEP_SECONDS variable into int, fallback to 10 seconds.")
+	}
+
+	fmt.Println("starting VirtletLogger")
```

Comment:
`glog.V(1).Info` there looks more correct. With that, it can be hidden/show on demand.
So by default there will be needed `-V=1` additional parameter added to binary invocation.